src/history/HistoryItem.tsx

```diff
@@ -11,7 +11,6 @@ type TProps = {
 };
 
 const HistoryItem: FC<TProps> = ({ data: { points, location, creationDate, title } }) => {
-  const { t } = useTranslation();
```

Comment:
Can the useTranslation hook still be used? Or not? 🤔

Reply:
yes!